app/controllers/concerns/vet360/writeable.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'common/exceptions/validation_errors'
 require 'va_profile/contact_information/service'
+require 'va_profile/profile_information/service'
 
 module Vet360
```

Comment:
Something to consider. I know this is working off of existing code and the intention might be to update as little as possible while moving things over to the new service, but I wonder if things would be simpler if this entire concern was removed and the controllers just call the models directly? The complexity was already there before this PR but it seems like this concern just complicates things without DRYing up much at all

Reply:
Yes, I agree. I will put this in the notes after this upgrade and we start to combine VAProfile services. This will no longer be needed.

Reply:
Same with transactionable.rb